src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -363,12 +369,11 @@
   otherwise use the default."
   [config   :- WebserverRawConfig
    num-cpus :- schema/Int]
-  (or (:max-threads config)
-      (let [calc-max-threads (max (calculate-required-threads config num-cpus)
-                                  default-max-threads)]
-        (log/warn "Thread pool size not configured so using a size of "
-                  calc-max-threads)
-        calc-max-threads)))
+  (let [max-threads (or (:max-threads config)
+                        (max (calculate-required-threads config num-cpus)
+                             default-max-threads))]
+    (log/debugf "Using webserver thread pool size of %d" max-threads)
```

Comment:
This came out of some informal discussions about the usefulness of the warn message.  Thought it might be better to have a way to see the number of threads that the server is going to use regardless of the configuration but only do it at a debug level since it's probably not as interesting for standard usage.